dali/python/setup.py.in

```diff
@@ -87,7 +87,7 @@ For more details please check the
           # the latest astunparse (1.6.3) doesn't work with any other six than
           # 1.16 on python 3.12 due to import six.moves
           'six >= 1.16, <= 1.16',
-          'dm-tree <= 0.1.8',
+          'dm-tree <= 0.1.9',
```

Comment:
nit, we have those now in [DALI](https://github.com/NVIDIA/DALI/tree/main)/[conda](https://github.com/NVIDIA/DALI/tree/main/conda)/[dali_python_bindings](https://github.com/NVIDIA/DALI/tree/main/conda/dali_python_bindings)/[recipe] (https://github.com/NVIDIA/DALI/tree/main/conda/dali_python_bindings/recipe)
/meta.yaml

too. Although, they seem to bound the version the other way round. :D

Reply:
That's right. Not sure why it's like that, but they have already picked `0.1.9` even without my update